app/models/pubmed_source_record.rb

```diff
@@ -323,14 +319,14 @@ def author_to_hash(author)
   # and https://dtd.nlm.nih.gov/ncbi/pubmed/doc/out/180101/att-PubStatus.html for PubStatus type definitions
   # and https://dtd.nlm.nih.gov/ncbi/pubmed/doc/out/180101/el-JournalIssue.html for the JournalIssue definition
   # and https://dtd.nlm.nih.gov/ncbi/pubmed/doc/out/180101/el-ArticleDate.html for the ArticleDate definition
-  def pubmed_date_xpaths(date_part)
+  def pubmed_date_xpaths
     [
-      "MedlineCitation/Article/Journal/JournalIssue/PubDate/#{date_part}",
-      "MedlineCitation/Article/ArticleDate/#{date_part}",
-      "PubmedData/History/PubMedPubDate[@PubStatus='accepted']/#{date_part}",
-      "PubmedData/History/PubMedPubDate[@PubStatus='pubmed']/#{date_part}",
-      "PubmedData/History/PubMedPubDate[@PubStatus='medline']/#{date_part}",
-      "PubmedData/History/PubMedPubDate[@PubStatus='entrez']/#{date_part}"
+      'MedlineCitation/Article/Journal/JournalIssue/PubDate',
+      'MedlineCitation/Article/ArticleDate',
+      "PubmedData/History/PubMedPubDate[@PubStatus='accepted']",
+      "PubmedData/History/PubMedPubDate[@PubStatus='pubmed']",
+      "PubmedData/History/PubMedPubDate[@PubStatus='medline']",
+      "PubmedData/History/PubMedPubDate[@PubStatus='entrez']"
     ]
```

Comment:
this method now makes more sense without passing in a parameter, since we now get the month and day together in one pass through this array and it makes it easier to do this way